boa/code/version.py

```diff
@@ -0,0 +1,30 @@
+from boa.code.compiler import Neo2Compiler, Neo3Compiler
+from boa.code.converter import Neo2Converter, Neo3Converter
+
+
+class Version:
```

Comment:
I don't feel that this is the best name to use, `Version.compiler.save` doesn't feel right, why `Version` has a compiler?
Maybe `Facade` or something else, idk